onto-viewer-web-app/config/ontology_config.xml

```diff
@@ -17,7 +17,8 @@
 
   <ontologyModuleIgnorePattern>^(About|Metadata).*</ontologyModuleIgnorePattern>
   <ontologyModuleIgnorePattern>http.*(About|Metadata).*</ontologyModuleIgnorePattern>
-
+  <ontologyModuleIgnorePattern>^(rdf-schema|rdf-syntax-ns).*</ontologyModuleIgnorePattern>
```

Comment:
Why do we need to ignore these?

Reply:
rdf-schema and rdf-syntax-ns (static ontologies) have "#" in the iri, which makes these ontologies display in the module tree.

Reply:
If we don't want to display an ontology as a module, we can use the `ontologyModuleToIgnore` config parameter with the ontology's IRI not to treat the ontology with this IRI as a module. E.g.

`<ontologyModuleToIgnore>http://www.w3.org/2000/01/rdf-schema#</ontologyModuleToIgnore>`

Reply:
> If we don't want to display an ontology as a module, we can use the `ontologyModuleToIgnore` config parameter with the ontology's IRI not to treat the ontology with this IRI as a module. E.g.
> 
> `<ontologyModuleToIgnore>http://www.w3.org/2000/01/rdf-schema#</ontologyModuleToIgnore>`

I think this is more flexible than just hard-coding removal of '#' char - the latter may produce some nasty surprises in the future.

Reply:
> `<ontologyModuleToIgnore>http://www.w3.org/2000/01/rdf-schema#</ontologyModuleToIgnore>`

using such a tag would be a good idea